### PR TITLE
Feature/keyboard

### PR DESCRIPTION
**Pull request by ericdachen**
Missing a lot of logic that needs to be mapped to the phrase board but this is kind of what I was going for where all the states are controlled in interface.jsx! Open to any and ALL feedback this is totally just a draft.

### CODE REVIEW

File: phrasle/src/components/interface/interface.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    //Logic left for later once we can map to the phraseboard component
+  }
+
+  function handleDelete(e) {
+    setGuess((guess) => guess.slice(0, -1));
+    console.log(guess);
+  }
+
+  const listTop = topRow.map((val) => (
+    <span key={val} data-key={val} onClick={() => handleKey(val)}>
```
Comment (alexthemanatee) [inline phrasle/src/components/interface/interface.jsx:32]: What are you using the data-key attribute here for?

Comment (alexthemanatee) [inline phrasle/src/components/interface/interface.jsx]: What are you using the data-key attribute here for?

File: phrasle/src/components/letterbox/key.jsx [hunk 2]
```diff
@@ -11,19 +11,46 @@ const Button = styled.button`
...
+`;
 
 function Key(props) {
-        return (
-            <Button>{props.letter}</Button>
-        );  
+  const [clicked, setClicked] = useState(false);
+  const [correct, setCorrect] = useState(props.phrase);
+
+  function handleClick(e) {
```
Comment (alexthemanatee) [inline phrasle/src/components/letterbox/key.jsx:37]: Do you ever want to unclick?

File: phrasle/src/components/interface/interface.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      <Key letter={val}></Key>
+    </span>
+  ));
+  const listMid = midRow.map((val) => (
+    <span key={val} data-key={val} onClick={() => handleKey(val)}>
+      <Key letter={val}></Key>
+    </span>
+  ));
+  const listBot = botRow.map((val) => (
```
Comment (alexthemanatee) [inline phrasle/src/components/interface/interface.jsx:41]: If you wanted to, you could split this "KeyRow" component into a separate file.

File: phrasle/src/components/interface/interface.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+function Interface() {
+  const [guess, setGuess] = useState("");
+
+  const topRow = ["Q", "W", "E", "R", "T", "Y", "U", "I", "O", "P"];
+  const midRow = ["A", "S", "D", "F", "G", "H", "J", "K", "L"];
+  const botRow = ["Z", "X", "C", "V", "N", "M"];
+
+  function handleKey(e) {
+    setGuess((guess) => guess.concat(e));
+    console.log(guess);
```
Comment (tailaiwang) [inline phrasle/src/components/interface/interface.jsx:19]: Remove logs from when you push

File: phrasle/src/components/letterbox/key.jsx (hunk 2 above)
Comment (tailaiwang) [reply phrasle/src/components/letterbox/key.jsx:37]: ^^

**Review by tailaiwang — APPROVED** [review]
Lgtm, just remove the console.logs

**Pull request merged**